index.css

```
@@ -1,997 +1,411 @@
-  -moz-osx-font-smoothing: grayscale;
-  line-height: 1.45;
-  overflow-x: hidden;
-}
-
-/* Utility Containers */
-.container{
-  width: calc(100% - 2rem);
-  max-width: var(--container);
-  margin: 0 auto;
-  padding: 0 1rem;
+  font-family: system-ui, -apple-system, Segoe UI, Roboto, sans-serif;
```

Stylelint (17.3.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)